.github/workflows/poeditor-export.yaml

```
@@ -0,0 +1,20 @@
+name: "Trigger POEditor Translations Export"
+
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - "[0-9]+.x"
+        paths:
+            - 'translations/admin.en.yaml'
```

Should we add `admin_ext.en.yaml` as well?
```suggestion
            - 'translations/admin.en.yaml'
            - 'translations/admin_ext.en.yaml'
```

---

Isn't this covered by the "extended" part in the csv?
https://github.com/pimcore/poeditor-export-action/pull/14/files#diff-3e8dca5aec3d81be5ce92295a045dea8f99d41b34caa0bc278e750e56995fbc5R21

But I am not sure either

---

Still not sure how this is handled